### ISSUE TITLE
May I ask how much of the targets_u shape?

### ISSUE THREAD

**Issue by woailunhua** [yasminhossam/fixMatchSeg-Muc#1]
targets_u = preds_u_w.squeeze(1).clone().detach()
        preds_label = preds_u_w.squeeze(1).clone().detach()
        preds_label[preds_label>0.5]=1
        preds_label[preds_label<=0.5]=0

        bg = (targets_u<=0.5)*targets_u #background
        fg = (targets_u>0.5)*targets_u  #foreground
        pixel_mean_bg = torch.true_divide(bg.sum((1,2)),(bg!=0).sum((1,2)))
        pixel_mean_fg = torch.true_divide(fg.sum((1,2)),(fg!=0).sum((1,2)))
        mask = (pixel_mean_fg>=0.9)|(pixel_mean_bg<=0.1)

May I ask how much of the targets_u shape?
